support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -200,7 +220,8 @@ function userInTest(
 }
 
 function randomNumber(mvtId: number, seed: number): number {
-	const rng = seedrandom(mvtId + seed);
+	const numberToSeed = mvtId + seed;
+	const rng = seedrandom(numberToSeed.toString());
```

Comment:
I've added `@types/seedrandom` and this `seedrandom` function takes a `string`.

Reply:
There's a PR in flight [here](https://github.com/guardian/support-frontend/pull/3324#discussion_r748232270) which modifies this file that also caused the build to fail

Reply:
Specific comment [here](https://github.com/guardian/support-frontend/pull/3324#discussion_r748232270)

Reply:
@marialani thanks for drawing my attention to this, I've left a comment